src/index.js

```diff
@@ -32,6 +32,7 @@
     var currentlyDefinedFactory;
     var currentlyDefinedModule;
     var toString = loaded.toString;
+    var head = doc.getElementsByTagName('head')[0];
```

Comment:
We definitely can use `querySelector` which is shorter, but also do you know any reason why we can't use `doc.head`? It seems to be available on every target browser...